getting-started-dao.adoc

```diff
@@ -0,0 +1,214 @@
+* There's a 2 BSQ fee to make a proposal to discourage spam.
+
+Once you're done making a proposal, you can sit back and relax until voting starts.
+
+=== 2. Vote on proposals
+
+[.float-group]
+--
+[.right.text-center]
+.Cast vote on a proposal.
+image::vote-proposal.png[Cast vote on proposal,400,400]
+
+Once the period to make proposals is over, it's time to vote. As long as you have some BSQ, you can vote—it doesn't matter if you submitted a proposal or not.
+
+You can see all proposals in the current voting cycle in the `Browse open proposals` section of the `Governance` tab on the `DAO` screen.
+
+Click through each proposal to see more details, and then vote on the proposals you want—you can vote for the ones you like (ideally, the ones you know intimately and have a strong opinion about).
+
+--
+
+[.float-group]
+--
+[.left.text-center]
+.Select voting weight and submit all votes.
+image::vote-submit.png[Select voting weight ad submit all votes,400,400]
+
+When you're done casting your votes, make sure you actually submit your votes by clicking the `Vote on all proposals` button at the bottom of the screen.
+
+To submit your votes, you'll need to specify a BSQ amount to use for **voting weight**. The more BSQ you include in your vote, the more weight the vote carries. This BSQ cannot be traded or otherwise transferred for the remainder of the voting cycle, or your vote will be rendered invalid. It will be released for you to use again as soon as the voting cycle has finished.
+
+This is why, as you see in the screenshot to the left, we use most (but not all) of our BSQ for the voting weight—we left some BSQ available for spending in case we need it for other purposes during the voting cycle.
+
+--
+
+`Vote weight from earned BSQ` is BSQ earned from contributing to Bisq: this BSQ is treated slightly differently from BSQ bought on the open market when determining a stakeholder's voting weight.
+
+For an overview of voting weight, see <<user-dao-intro#determine-strategy,this section>> of our Introduction to the Bisq DAO doc.
+
+=== 3. Reveal your vote
+
+After the blind voting phase is over, **you must make sure Bisq goes back online at some point during the vote reveal phase to publish your vote reveal transaction**.
+
+IMPORTANT: Bisq will publish the transaction automatically when it's online, so you don't have to actually do anything, but you can check the https://explorer.bisq.network/[BSQ block explorer^] to be sure the transaction was published. More details <<#explore-a-bsq-block-explorer,below>>.
+
+This transaction includes the key to decrypt your vote from the last phase. It also includes a representation of all votes you've received from the network, which is necessary to determine which peer on the network has the most complete view of all votes cast. See more details <<dao/specification#vote-reveal-phase,here>>.
+
```

Comment:
"which is necessary to determine which peer on the network has the most complete view of all votes cast" -Maybe that is a bit too difficult to understand. what about:
which is necessary to determine a consensus about the published votes in the network